hub/powertoys/run.md

```diff
@@ -23,80 +23,74 @@ To use PowerToys Run, select <kbd>Alt</kbd>+<kbd>Space</kbd> and start typing! _
 
 PowerToys Run features include:
 
-- Search for applications, folders, or files
+- Search for applications, folders or files
 - Search for running processes (previously known as [WindowWalker](https://github.com/betsegaw/windowwalker/))
 - Clickable buttons with keyboard shortcuts (such as _Open as administrator_ or _Open containing folder_)
 - Invoke Shell Plugin using `>` (for example, `> Shell:startup` will open the Windows startup folder)
 - Do a simple calculation using calculator
-- Executing system commands
-- Getting time and date information
-- Converting units
-- Opening web pages or starting a web search
+- Execute system commands
+- Get time and date information
+- Convert units
+- Open web pages or start a web search
 
-## Settings
 
-The following general options are available for PowerToys Run in the PowerToys settings menu.
+## PowerToys Settings
 
-| Settings | Description |
+The following general options are available on the PowerToys Settings page.
+
+| Setting | Description |
 | :--- | :--- |
-| Activation shortcut | Define the keyboard shortcut to open/hide PowerToys Run |
-| Use centralized keyboard hook | You can try this setting if there are issues with the shortcut 
+| Activation shortcut | Define the keyboard shortcut to show/hide PowerToys Run |
+| Use centralized keyboard hook | Try this setting if there are issues with the shortcut
 | Ignore shortcuts in Fullscreen mode | When in full-screen (F11), PowerToys Run won't be engaged with the shortcut |
 | Maximum number of results | Maximum number of results shown without scrolling |
 | Clear the previous query on launch | When launched, previous searches will not be highlighted |
 | Preferred monitor position | If multiple monitors are in use, PowerToys Run can be launched on the desired monitor:<br />- Primary display<br />- Display with mouse cursor<br />- Display with focused window |
 | App theme | Change the color theme used by PowerToys Run |
 
 
-## Usage
-
-### Keyboard shortcuts
+## Keyboard shortcuts
 
-| Shortcuts | Action |
+| Shortcut | Action |
 | :--- | :--- |
-|<kbd>Alt</kbd>+<kbd>Space</kbd> (default) | Open or hide PowerToys Run |
+|<kbd>Alt</kbd>+<kbd>Space</kbd> (default) | Show or hide PowerToys Run |
 |<kbd>Esc</kbd> | Hide PowerToys Run |
 |<kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>Enter</kbd> | Open the selected application as administrator (only applicable to applications) |
 |<kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>U</kbd> | Open the selected application as different user (only applicable to applications) |
 |<kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>E</kbd> | Open containing folder in File Explorer (only applicable to applications and files) |
 |<kbd>Ctrl</kbd>+<kbd>C</kbd> | Copy path location (only applicable to folders and files) |
-|<kbd>Tab</kbd> | Navigate through the search result and context menu buttons |
+|<kbd>Tab</kbd> | Navigate through the search results and context menu buttons |
 
 
-### Direct activation commands
+## Plugins
```

Comment:
I don't like this change as it was more structured before.

And by the way. The plugins without keyword are not mentioned in the table.

Reply:
> I don't like this change as it was more structured before.

Sometimes we think the same things, here apparently we don't. I didn't like the current/prevous style. To me, this hierarchy makes sense.

> And by the way. The plugins without keyword are not mentioned in the table.

That's not on me :)

Reply:
>> And by the way. The plugins without keyword are not mentioned in the table.

> That's not on me :)

I know that. But your headers let the user think the table is a full list of all plugins. With the old layout it was clear that the topic of the table are the action keys and not the list of all plugins.

Reply:
I think we should readd the header "direct activation commands" after the text that we have the plugin system. This makes sense as we have this text and tipp and table that are primarily cover the topic direct activation commands.

Reply:
Hm, Thought I had a good idea. But if you insist on this table being focussed on the DAC's, I will change it back. Note: I will be removing the column "enabled by default" then, because that has nothing to do with the DAC.